flyteplugins/go/tasks/plugins/k8s/ray/config.go

```diff
@@ -22,7 +22,6 @@ var (
 		IncludeDashboard:         true,
 		DashboardHost:            "0.0.0.0",
 		EnableUsageStats:         false,
-		KubeRayCrdVersion:        "v1alpha1",
```

Comment:
cc @peterghaddad

Reply:
@pingsutw after this MR is complete is there no going back to v1alpha1?

Reply:
Unsure if Flyte has decided to do a hard switchover from v1alpha1 to v1. If not, then I would recommend with staying on v1alpha1 as the default value for the time being.

Reply:
v1alpha1 in kuberay 1.0.0 and kuberay 1.1.0 are different. That being said, if you upgrade flyte but keep kuberay at 1.0.0. The new v1alpha1 imported in flyte cannot be run on the old kuberay. Also, there is no purpose to keep both v1alpha1 and v1, so we decide to discard v1alpha1.

If users really want to stay at v1alpha1, they can feel free to use the old flyte and kuberay 1.0.0. But if they want to upgrade kuberay to 1.1.0, it is required for them to upgrade flyte as well.

Reply:
Users can stills stay at v1.0.0 with old flyte server. See (old, old) row
<img width="654" alt="image" src="https://github.com/flyteorg/flyte/assets/24364830/19fa9d5f-2842-47cd-8e68-42b1c25399b6">